### DISCUSSION TITLE
Condizionatore riconosciuto da google assistant come termostato

### DISCUSSION THREAD

**Discussion by luxbarba** [indomus/forum#622] [HUB: Home Assistant]
Buongiorno a tutti. Possiedo 4 climatizzatori wifi della toyotomi che ho inserito in HA tramite l'integrazione Gree A/C e funziona tutto perfettamente. Per quanto riguarda l'integrazione in google assistant, prima la facevo tramite la loro app nativa che funziona malisssimo (I condizionatori pur essendo connessi alla rete wifi spesso mi si disconnettono dall'app nativa) anche se venivano riconosciuti correttamente come condizionatori da google assistant. Per questo motivo ho deciso di integrarli in google assistant tramite HA. Il problema è che vengono riconosciuti da google assistant come termostati e questo comporta il fatto che non si possono accendere il modalità freddo...Ho provato anche a creare delle routine sull'app di Google Home ma quando setto per esempio la temperatura del termostato a 27 gradi mi dice che non posso farlo.... C'è un modo per farli riconoscere a google assistant come condizionatori? di seguito riporto la mia configurazione su HA con mac address oscurato:
```climate:
   - platform: gree
     name: Climatizzatore Cucina
     host: 192.168.1.5
     port: 7000
     mac: xx:xx:xx:xx:xx:xx
     target_temp_step: 1
   
   - platform: gree
     name: Climatizzatore Studio
     host: 192.168.1.38
     port: 7000
     mac: xx:xx:xx:xx:xx:xx
     target_temp_step: 1
   
   - platform: gree
     name: Climatizzatore Letto
     host: 192.168.1.122
     port: 7000
     mac: xx:xx:xx:xx:xx:xx
     target_temp_step: 1
     
   - platform: gree
     name: Climatizzatore Soggiorno
     host: 192.168.1.244
     port: 7000
     mac: xx:xx:xx:xx:xx:xx
     target_temp_step: 1
```
questo invece è il codice per l'esposizione dei climatizzatori su google assistant:
```google_assistant:
  project_id: mio_progetto
  service_account: !include home-assistant-xxxxx-xxxxxxxxxxxx.json
  report_state: true
  exposed_domains:
    - climate
```

**Comment by snakuzzo** [comment]
https://www.home-assistant.io/integrations/google_assistant/#climate-operation-modes

**Reply by luxbarba** [reply]
Si già lo avevo letto. Non ho capito come lo devo scrivere.....

**Reply by snakuzzo** [reply]
cosa vorresti scrivere ? Il link è per farti capire che di default le modalità operative ci sono.
Se non viene loggato nulla a debug, prova ad aprire una issue sul progetto di [core](https://github.com/home-assistant/core/issues) (magari vedi se ci sono già cose simili)

**Reply by luxbarba** [reply]
Sicuramente non mi sono saputo spiegare. Pensavo che ci fosse una stringa da inserire per far riconoscere a google assistant il condizionatore come "condizionatore" e non come termostato.

**Reply by snakuzzo** [reply]
il condizionatore, così come un termotato è un `climate`
